test/parallel/test-tls-failed-handshake-emits-error.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+var common = require('../common');
+
+if (!common.hasCrypto) {
+  common.skip('missing crypto');
+  return;
+}
+var tls = require('tls');
+var net = require('net');
+
+var bonkers = Buffer.alloc(1024, 42);
+
+var server = net.createServer(common.mustCall(function(c) {
+  setTimeout(common.mustCall(function() {
```

Comment:
we don't need common.mustCall here, but it does not hurt either. Any other opinions?